frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/StaticEmbedSetupPane.tsx

```diff
@@ -322,7 +322,7 @@ export const StaticEmbedSetupPane = ({
                     hidden={activePane !== "preview"}
                     className={CS.flexFull}
                     previewUrl={iframeUrl}
-                    isTransparent={displayOptions.theme === "transparent"}
+                    isTransparent={!displayOptions.background}
```

Comment:
This flag is used whether or not to display the checkerboard background.
![image](https://github.com/metabase/metabase/assets/1937582/7f6d3fa3-2284-445f-b854-2e44b3d3749f)